store/CSDK/test/main.cpp

```diff
@@ -709,6 +709,7 @@ bool testWithTableProperty(JNIEnv *env, char *path, int argc, char **argv) {
         writer.outputPath(path);
         writer.withCsvInput(jsonSchema);
         writer.withTableProperty("sort_columns", "shortField");
+        writer.enableLocalDictionary(false);
```

Comment:
Can you test if there is any possibility to give NULL in C and how it is sent to java layer ? true or false?

Reply:
add test code for it. In CPP, it will convert NULL to false when user invoke CarbonWriter::enableLocalDictionary.